lemarche/www/siaes/forms.py

```diff
@@ -22,6 +22,16 @@
     ("ZRR", "Zone de revitalisation rurale (ZRR)"),
 )
 
+FORM_CA_CHOICES = (
+    ("", ""),
+    ("-100000", "Moins de 100K€"),
```

Comment:
```suggestion
    ("-100000", "Moins de 100 K€"),
```
dans `tenders/constants.py` on a un autre exemple de champ / filtre qui contient des `K€`. On avait fait le choix de mettre un espace entre le chiffre et l'unité, peut-être homogénéiser dans ce sens ?